test/e2e/catalog_e2e_test.go

```diff
@@ -6,6 +6,7 @@ package e2e
 import (
 	"context"
 	"fmt"
+	operatorsv1 "github.com/operator-framework/api/pkg/operators/v1"
```

Comment:
nit: go imports? Also the package imports seem to have shifted rightwards (looking at the diff) so maybe gofmt as well?

Reply:
Yeah, I saw this too. I believe we ignore the test/e2e package in the goreleaser configuration (too many changes needed), but we could configure the importas linter to enforce this package import like we do in the rukpak repository.

Reply:
oh whoops - I was using my IDE which I guess resorted the imports...